vgm_trim.c

```diff
@@ -50,18 +53,25 @@ int main(int argc, char* argv[])
 	argbase = 1;
 	OptsTrim = 0x00;
 	OptsWarn = 0x00;
+	OptsOverwrite = 0x00;
 	while(argbase < argc && argv[argbase][0] == '-')
 	{
 		if (! stricmp(argv[argbase] + 1, "help"))
 		{
-			printf("Usage: vgm_trim [-state] [-nonotewarn] File.vgm\n");
+			printf("Usage: %s [-state] [-nonotewarn] [-o] File.vgm\n",argv[0]);
 			printf("                StartSmpl LoopSmpl EndSmpl [OutFile.vgm]\n");
 			printf("\n");
 			printf("Options:\n");
 			printf("    -state: put a save state of the chips at the start of the VGM\n");
 			printf("    -NoNoteWarn: don't print warnings about notes playing at EOF\n");
+			printf("    -o: overwrite even if OutFile.vgm exists\n");
```

Comment:
Someone on IRC just reminded me that requiring an additional parameter to enforce overwriting:
1. is a feature change, for which a separate PR would be good
2. is actively breaking the workflow of a some very active vgmrips contributors, which is "drop the original file on vgm_trim.exe and enter new values to fix the trimming"

So please:
- put this into a separate PR
- turn the option into a "don't overwrite" so that the default behaviour does not change
  Overwriting by default is also how common Unix tools like `cp` and `mv` work.

Thanks!

Reply:
No problem. Frankly speaking, I believe a good program should never overwrite any user's file without user's consent, unless it is explicitly specified by user (e.g. --always-overwrite).